docs/book/src/logos/README.md

```diff
@@ -0,0 +1,11 @@
+# KubeBuilder Logos
+
+The official location for the logos is in a [public GCS
+bucket][kb-logos-gcs].
+
+These logos are copies used in the book, resized to their appropriate
+sizes. They're tracked with Netlify's *Large Media Storage*, which is
+a git-lfs implementation. Please do not try and use GitHub's git-lfs
+implementation to store these.
+
+[kb-logos-gcs]: https://console.cloud.google.com/storage/browser/kubebuilder-logos
```

Comment:
This requires users to sign in with a GCP account.
Maybe better to use https://storage.googleapis.com/kubebuilder-logos/.